src/test/java/mate/academy/dao/impl/RoleDaoImplTest.java

```diff
@@ -0,0 +1,69 @@
+package mate.academy.dao.impl;
+
+import mate.academy.dao.RoleDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.model.Role;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RoleDaoImplTest extends AbstractTest {
+    private static Role adminRole;
+    private static Role userRole;
+    private RoleDao roleDao;
+
+    @Override
+    protected Class<?>[] entities() {
+        return new Class[]{Role.class};
+    }
+
+    @BeforeAll
+    static void setRoleDao() {
```

Comment:
```suggestion
    static void init() {
```

Reply:
I change to initRole, because have inconsistent with method init AbstractTest.class